src/raven.js

```diff
@@ -1134,7 +1179,14 @@ Raven.prototype = {
               // preserve arity
               try {
                 if (fn && fn.handleEvent) {
-                  fn.handleEvent = self.wrap(fn.handleEvent);
+                  fn.handleEvent = self.wrap(
+                    {
+                      mechanism: {
+                        extra: {target: global, function: 'handleEvent', handler: fn.name}
```

Comment:
Isn't this too much if we store references to `global` in the mechanism?

Reply:
That's just a string